tests/dromaeo/dromaeo

```diff
@@ -0,0 +1 @@
+Subproject commit 0df04c071f49dce9ecf578c4e152706aacc886f1
```

Comment:
This file is produced during `./mach test-dromaeo`. I think we should put it in .gitignore.